### ISSUE TITLE
Guard notifier always expects 2 arguments while sprockets sends one 

### ISSUE THREAD

**Issue by BSierakowski** [guard/guard#737]
Recently upgraded to rails 4 and got the following guard error:

```
19:33:25 guard.1  | 19:33:25 - ERROR - Guard::Sprockets failed to achieve its <start>, exception was:
19:33:25 guard.1  | > [#C5DBB934BE1A] ArgumentError: wrong number of arguments (1 for 2)
```

From the file
```
gems/guard-2.11.1/lib/guard/notifier.rb:26:in `notify'
```

I was able to patch locally by giving a default hash if the caller doesn't pass one thru, ala:

```
def self.notify(message, options={})
```

Am I somehow using an old version of something that isn't sending options?

**Comment by e2** [comment]
This was fixed on master - I've just released Guard v2.12.0 with the fix.

(Sorry for the wait).

**Comment by BSierakowski** [comment]
Awesome, thank you!
